### DISCUSSION TITLE
Image peek_pixels  always return None

### DISCUSSION THREAD

**Discussion by toplinuxsir** [rust-skia/rust-skia#673] [Q&A] [answered]
example code as below 
```
 let file_data = fs::read(file_name)?;

    //let file_data = include_bytes!(file_name);

    let data = Data::new_copy(&file_data);

// assert file_data.len () > 0

    let img = Image::from_encoded(data).unwrap();

    let pixmap_t = img.peek_pixels(); // here peek_pixels always return None
 
```

**Comment by pragmatrix** [comment, marked as answer]
For `peek_pixels`, the `Image` must be internally represented as a pixel array. To convert an `Image` to a pixel array, use the `.new_raster_image()` function after you've decoded the image.

Alternatively, use `read_pixels()`.

**Comment by toplinuxsir** [comment]
@pragmatrix Hi, Thanks 

I think ,It's better to rename new_raster_image  to to_raster_image

```
 let file_data = fs::read(file_name)?;


    let data = Data::new_copy(&file_data);

let img = Image::from_encoded(data).unwrap();
let image = img.new_raster_image().unwrap();

let pixmap = image.peek_pixels();
```


Thanks !

**Reply by pragmatrix** [reply]
> ,It's better to rename new_raster_image to to_raster_image

Yes, thank you, you are right, I thought about that when I was going through the binding code, too. Will do a PR soon and deprecate the old functions.
